source/install/desktop.rst

```diff
@@ -43,11 +43,15 @@ then open the Group Policy Management Console and you should see under "Computer
 
 Following Group Policies are available:
 
-+------------------------+-----------------------------------------------------+----------------------+
-| Group Policy           | Description                                         | Required Version     |
-+========================+=====================================================+======================+
-| Disable Auto Updater   | Controls the behavior of the internal Autoupdater   | 4.3 or later         |
-+------------------------+-----------------------------------------------------+----------------------+
++--------------------------------------+---------------------------------------------------+----------------+----------------------+
+| Group Policy                         | Description                                       | Type of Policy | Required Version     |
++======================================+===================================================+================+======================+
+| Disable Auto Updater                 | Controls the behavior of the internal Autoupdater | Machine        | 4.3 or later         |
```

Comment:
Please be consistent with the spelling of "Auto updater"

Reply:
done :)